### DISCUSSION TITLE
[NEW PRIMER PUBLISHED]: Artificial Intelligence - AI Chipsets

### DISCUSSION THREAD

**Discussion by leomordasini** [finos/zenith#81] [Announcements]
### New Primer Published in the AI space!

Take a look at the following primer to get started
- <a href="https://github.com/finos/zenith/blob/main/primers/artificial-intelligence/deep-dives/AI%20Chipsets.md"> AI Chipsets Primer</a>
- <a href="https://github.com/finos/zenith/blob/main/primers/artificial-intelligence/deep-dives/AI%20Chipsets.md#call-to-action">AI Chipset: Call to Action</a>

Discuss with your peers here and collaborate. Once you have an idea you'd like to submit for a proposal, <a href="https://github.com/finos/zenith/issues/new?assignees=leomordasini&labels=idea&projects=&template=Idea_submission.yml&title=%5BIdea%5D%3A+">fill out this form</a>.

**Comment by Simon-phiri** [comment]
Hey gurus

**Reply by tenatzen** [reply]
Hey! I'm working on getting some more details up about all of the opportunities. Would love to get feedback on the opportunities identified in the primer
